release.md

```diff
@@ -48,7 +112,7 @@ The *driver version number* represents the build version of kernel-space drivers
 
 > _Note that `API_VERSION` and `SCHEMA_VERSION` are only used internally. On the other hand, only the **driver version number** will be used **to tag a new release**._
 
-## Libs versioning
+### Libs versioning
```

Comment:
While it is part of the Falco repo demystifying https://github.com/falcosecurity/falco/blob/master/userspace/engine/falco_engine_version.h#L19 here would be amazing or linking to a respective doc in the falco repo works as well.

```
FALCO_ENGINE_VERSION
FALCO_FIELDS_CHECKSUM
```

How does this relate to the driver or libs versioning?

Reply:
FALCO_ENGINE_VERSION does not directly link to the driver version. When an event field is introduced,  SCHEMA_VERSION and FALCO_ENGINE_VERSION need to be bumped.

Nevertheless, FALCO_ENGINE_VERSION can also be bumped for other reasons, like breaking changes introduced in the Falco engine that are not necessarily tied to the drivers or the libs.

Unfortunately, we can't easily change how FALCO_ENGINE_VERSION works since we know it is used in many rules files. But it's another topic. Would we open an issue in the Falco repo? Maybe a bit of documentation there will help.

Reply:
https://github.com/falcosecurity/falco/issues/2195